zulipterminal/themes/solarized_dark.py

```diff
@@ -0,0 +1,90 @@
+"""
+SOLARIZED DARK
+--------------
+
+
+"""
+
+from pygments.styles.solarized import SolarizedDarkStyle
+
+from zulipterminal.themes.colors_solarized import DefaultBoldColor as Color
+
+STYLES = {
+    # style_name       :  foreground                   background
+    None               : (Color.base0,                 Color.base03),
+    'selected'         : (Color.base0,                 Color.base02),
+    'msg_selected'     : (Color.base0,                 Color.base02),
+    'header'           : (Color.cyan,                  Color.blue),
+    'general_narrow'   : (Color.base0,                 Color.blue),
+    'general_bar'      : (Color.base0,                 Color.base03),
+    'name'             : (Color.red__BOLD,             Color.base03),
+    'unread'           : (Color.violet,                Color.base03),
+    'user_active'      : (Color.green,                 Color.base03),
+    'user_idle'        : (Color.yellow,                Color.base03),
+    'user_offline'     : (Color.base01,                Color.base03),
+    'user_inactive'    : (Color.base01,                Color.base03),
+    'title'            : (Color.base0__BOLD,           Color.base03),
+    'column_title'     : (Color.base0__BOLD,           Color.base03),
+    'time'             : (Color.base01,                Color.base03),
+    'bar'              : (Color.base0,                 Color.base00),
```

Comment:
According to the solarized spec, we shouldn't have `base00` in the dark theme? This would be more obvious if we stick to meaningful names like `background`, `foreground` and so on with the separate enum approach I suggest above.

Reply:
yeah, I will look into it